romancal/flatfield/tests/test_flatfield.py

```diff
@@ -54,3 +54,38 @@ def test_flatfield_step_interface(instrument, exptype):
     assert (result.data == wfi_image.data).all()
     assert result.var_flat.shape == shape
     assert result.meta.cal_step.flat_field == 'COMPLETE'
+
+
+
+@pytest.mark.parametrize(
+    "instrument, exptype",
+    [
+        ("WFI", "WFI_IMAGE"),
+    ]
+)
+@pytest.mark.skipif(
+    os.environ.get("CI") == "true",
+    reason="Roman CRDS servers are not currently available outside the internal network"
+)
+def test_crds_temporal_match(instrument, exptype):
+    """Test that the basic inferface works for data requiring a FLAT reffile"""
+
+    wfi_image = testutil.mk_level2_image(arrays=True)
+    wfi_image.meta.instrument.name = instrument
+    wfi_image.meta.instrument.detector = 'WFI01'
+    wfi_image.meta.instrument.optical_element = 'F158'
+
+    wfi_image.meta.observation.start_time = Time('2020-01-01T11:11:11.110')
+    wfi_image.meta.observation.end_time = Time('2020-01-01T11:33:11.110')
+
+    wfi_image.meta.exposure.type = exptype
+    wfi_image_model = ImageModel(wfi_image)
+
+    step = FlatFieldStep()
+    reffile = step.get_reference_file(wfi_image_model, "flat")
+    assert ("/".join(reffile.rsplit("/", 3)[1:]) == "roman/wfi/roman_wfi_flat_0057.asdf")
+
+    wfi_image_model.meta.observation.start_time = Time('2021-08-11T11:11:11.110')
+    wfi_image_model.meta.observation.end_time = Time('2021-08-11T11:33:11.110')
+    reffile = step.get_reference_file(wfi_image_model, "flat")
+    assert ("/".join(reffile.rsplit("/", 3)[1:]) == "roman/wfi/roman_wfi_flat_0039.asdf")
```

Comment:
This path depends on how the cache is configured -- the per-instrument directory structure here is less common than the "flat" structure that does not include the instrument name in the path.  I recommend comparing the filename only, something like:

```python
assert Path(reffile).name == "roman_wfi_flat_0039.asdf"
```

Reply:
Okay, if we keep the explicit filename check, I can simplify this.. I only specified it to easily show that the file we matched to was roman & wfi.. but, I that could be overkill.